h/tasks/indexer.py

```diff
@@ -42,6 +42,12 @@ def sync_annotations(limit):
     )
 
 
+@celery.task
+def delete_annotations_from_elasticsearch(limit):
+    """Delete annotations that've been deleted from the DB from Elasticsearch."""
+    celery.request.find_service(name="search_index").delete_annotations(limit)
```

Comment:
Rather than adding a new task, I'll probably add delete support into the existing sync annotations task.